doc/QUICKLOOK/snr_qa.rst

```diff
@@ -0,0 +1,59 @@
+====================
+qa-CAMERA-EXPID.yaml
+====================
+
+:Summary: This QA for QuickLook first includes the calculation of 
+        signal-to-noise in the spectrum after sky subtraction. 
+:Naming Convention: ``qa-{ARM}{SPECTROGRAPH}-{EXPID}.yaml``, where 
+        {ARM} is the 1-char arm name (r,b,z), {SPECTROGRAPH} indexes 
+        CCDs 0-9 on that arm, and {EXPID} is the 8-digit exposure ID.  
+        Together, {ARM}{SPECTROGRAPH} specify a {CAMERA}.
+:Regex: ``qa-[brz][0-9]-[0-9]{8}\.yaml``
+:File Type:  YAML
+
+
+Inputs
+======
+
+Written by qa_quicklook.py, with Calculate_SNR using:
+
+- frame
+- skymodel
+
+Contents
+========
+
+========== ================ ===========================
+KEYNAME    Type             Contents
+========== ================ ===========================
+SNR        Py dictionary    Spectral signal-to-noise
+========== ================ ===========================
+
+
+
+Dictionary Contents
+===================
+
+KEYNAME = SNR
+
+Spectral signal-to-noise calculations per fiber and per CCD amp.
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+================ ============= ========== ============
+KEY              Example Value Type       Comment
+================ ============= ========== ============
+ARM              r             char       Spectrograph arm b,r,z

```

Comment:
I just took a look at the raw .rst file & it is properly aligned, so don't worry about this.